CHANGELOG.md

```diff
@@ -1,4 +1,25 @@
 
+7.2.0 (December 2024)
+---------------------
+
+- "Interactive" Inset panels ([#549](https://github.com/ome/omero-figure/pull/549))
+- Use timeIncrement for Time if no plane-info metadata (Thanks to Jens Wendt) ([#589](https://github.com/ome/omero-figure/pull/589))
+- Add Hilo Checkbox for LUT Toggling (Thanks to Tehmina Enayat) ([#583](https://github.com/ome/omero-figure/pull/583))
+- Support for 'Points' in ROI functionality ([#555](https://github.com/ome/omero-figure/pull/555))
+- Add link from Images to Figure in the form of Key-Value pairs (Thanks to Rémy Dornier) ([#584](https://github.com/ome/omero-figure/pull/584))
+- Support for ROI Fill Colour (Thanks to Rémy Dornier) ([#577](https://github.com/ome/omero-figure/pull/577))
+- Panel Borders (Thanks to Rémy Dornier) ([#578](https://github.com/ome/omero-figure/pull/578))
+- Horizontal and Vertical image flipping (Thanks to Tehmina Enayat) ([#591](https://github.com/ome/omero-figure/pull/591))
+- Dynamic loading of Look-up Tables (Thanks to Tom Boissonnet) ([#596](https://github.com/ome/omero-figure/pull/596))
+- Rotate image panels by 90° (Thanks to Rémy Dornier) ([#595](https://github.com/ome/omero-figure/pull/595))
+- File Open filtering by Group & Owner improvements ([#592](https://github.com/ome/omero-figure/pull/592))
+
+Fixes:
```

Comment:
https://github.com/ome/omero-figure/pull/603 is missing

Reply:
Yeah, but that's just a follow-up PR fixing bugs that were missed from #549, not bugs that have been released. So it's kinda meaningless to list it.

Reply:
maybe add the 2PRs in that case

Reply:
OK, done. All good to go now?